R/pkg/R/SQLContext.R

```diff
@@ -149,6 +149,26 @@ createDataFrame <- function(sqlContext, data, schema = NULL, samplingRatio = 1.0
   dataFrame(sdf)
 }
 
+#' Create a DataFrame from an RDD
+#'
+#' Converts an RDD to a DataFrame by infer the types.
```

Comment:
Could you remove the reference to RDD in the comments here (and in createDataFrame). You could just make it `Converts R data.frame or list into DataFrame`. 

In the same spirit could you use `iris` or something like that in the example (instead of the lapply)